openedx/frontend-app-course-authoring
# fix: update mapToStateProps to match changes in TinyMceWidget

## Description

This PR fixes pages throwing errors when the `WysiwygEditor` used due to the redux state variable used in the child `TinyMceWidget`.

## Supporting information

JIRA Ticket: [TNL-11665](https://2u-internal.atlassian.net/browse/TNL-11665) 🔒 
>The Updates feature in Studio (content > Updates) is giving an “unexpected application error” when course teams click the “+ New update” button. This is blocking course teams from communicating with their learners.

## Testing instructions

1. Navigate to the Course Updates page
2. Confirm that existing updates and handouts load
3. Click "+ New update"
4. Confirm that editor loads
5. Add content and click "Post"
6. Click the edit button for an update
7. Confirm that the editor loads
8. Click "Cancel"
9. Click the edit button for handouts
10. Confirm that the editor loads
11. Click "Cancel"
12. Navigate to the Schedule and details page
13. Confirm that the page loads as expected


## Other information

This PR is dependent on frontend-lib-content-components [#489](https://github.com/openedx/frontend-lib-content-components/pull/489). The package upgrade generated by [#489](https://github.com/openedx/frontend-lib-content-components/pull/489) will be included in this PR
